inst/qml/msaGaugeLinearity.qml

```diff
@@ -81,14 +80,14 @@ Form
 
 		CheckBox
 		{
-			name:								"LBgraph"
+			name:								"linearityAndBiasPlot"
 			label:								qsTr("Linearity and bias graph")
 			checked:							true
 		}
 
 		CheckBox
 		{
-			name: 								"LBpercentGraph"
+			name: 								"percentageOfProcessVariationPlot"
```

Comment:
is there a reason to not call it `perceptProcessVariationPlot` since that's what the label is? Or the other way around, why not then change the label accordingly? :)

Reply:
I thought "percentage of" was even clearer than "percent". I'll have to double-check what makes most sense, but I'll make sure they are consistent.

Reply:
ok!